### DISCUSSION TITLE
AO 5.0

### DISCUSSION THREAD

**Discussion by geronimopilot** [kubilus1/autoortho#308] [General]
Autoortho is and has been a game changer - BRILLIANT! Thanks for all your hard work!!!!

Currently, I am running an older version of autoortho, (which seems to work perfectly),  and would like to know if 5.0 is out of beta? I am not the most computer savvy person and don't want to rock the boat - it seems that the 5.0 install is unlike earlier versions and I am not understanding what to do with the older files in custom scenery, and where to place all of the 5.0 components. 

Thank you!
Scott

**Comment by drmichaelgorski** [comment]
@kubilus1 Can you please release a new version of authoortho?
